products/idn/tools/sdk/index.md

```diff
@@ -0,0 +1,21 @@
+---
+id: sdk
+title: SDKs
+pagination_label: SDKs
+sidebar_label: SDKs
+sidebar_position: 1
+sidebar_class_name: sdk
+keywords: ['sdk']
+description: Learn how to use the Golang SDK in this guide. 
+slug: /tools/sdk
+tags: ['SDK']
+---
+
+## SDKs
```

Comment:
@james-haytko-sp are we going to go any deeper on this page? I'm not sure, just asking for posterity.

Reply:
I added a bit more to the description. I think maybe icons for the different SDKs would be good.